icons/key.svg

```diff
@@ -9,7 +9,7 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <circle cx="7.5" cy="15.5" r="5.5" />
+  <path d="m15.5 7.5 2.293 2.293a1 1 0 0 0 1.414 0l2.086-2.086a1 1 0 0 0 0-1.414L19 4" />
```

Comment:
The path should be optimized a bit. This doesn't affect the look
```suggestion
  <path d="m15.5 7.5 2.3 2.3a1 1 0 0 0 1.4 0l2.1-2.1a1 1 0 0 0 0-1.4L19 4"/>
```

Reply:
Agree +1